pyodbc.cpython-39-x86_64-linux-gnu.so,sha256=OuqOrfNvu3-_E2N6hZKQHJHhsyGwUZn5mRTtKuSg3vY,907553
pyodbc.pyi,sha256=SXkDoEs1fp-Ke4zaihF7fMoKi5LkfCwyAQ8f9FQ-vQg,32351
pyodbc-5.1.0.dist-info/WHEEL,sha256=uQ9JcPdAMEhlCRnfrhg6ydnbrPot9NaB8igB3QVnPX0,148
pyodbc-5.1.0.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-5.1.0.dist-info/METADATA,sha256=bFvJAkgfYisUwEBHrj722VY1z6mhdDRMFINyu6DVRnQ,2717
pyodbc-5.1.0.dist-info/top_level.txt,sha256=Rb69vS-SNN_Bfh99vX7eCcvEAJoyzowG3u4R7W36Flw,8
pyodbc-5.1.0.dist-info/RECORD,,
